### PR TITLE
 Fix for bug #1044639 : Implement "Delete tag"

### PR DESCRIPTION
**Pull request by sagarghuge**

### CODE REVIEW

File: GTG/gtk/browser/deletetags_dialog.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+# -----------------------------------------------------------------------------
+
+
+from gi.repository import Gtk
+
+from GTG import _, ngettext
+from GTG.gtk.browser import GnomeConfig
+
+
+class DeleteTagsDialog():
```
Comment (izidormatusov) [inline GTG/gtk/browser/deletetags_dialog.py]: Please, make it inherit from `object`:

```python
class DeleteTagsDialog(object):
```

**Comment by khurshid-alam** [comment]
@sagarghuge Has this merged into master?

**Comment by sagarghuge** [comment]
Nope

On Nov 9, 2016 8:46 PM, "Khurshid Alam" <notifications@github.com> wrote:

@sagarghuge <https://github.com/sagarghuge> Has this merged into master?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/getting-things-gnome/gtg/pull/26#issuecomment-259600765>,
or mute the thread

**Comment by khurshid-alam** [comment]
Any specific reason for that? It should still work with 3.1....no?

**Comment by sagarghuge** [comment]
I have to check actually. It's been a long time I haven't touched the code
base. I will check and update on this .

**Comment by khurshid-alam** [comment]
@sagarghuge Thank you. :) I will try to use/merge your code in 3.1 and see if it throws any error.

**Comment by diegogangl** [comment]
Tested this one and it works fine. The dialog needs to be updated to the current style, but that can be done after merging. @ploum see anything that should be fixed?

**Comment by diegogangl** [comment]
It's never too late to merge! Thanks for your work @sagarghuge

**Pull request merged**